tests/message/unittest_message.py

```diff
@@ -11,13 +11,13 @@
 
 def test_new_message(message_definitions: ValuesView[MessageDefinition]) -> None:
     def build_message(
-        message_definition: MessageDefinition, location_value: MessageLocationTuple
+        message_definition_: MessageDefinition, location_value: MessageLocationTuple
```

Comment:
?
Is this interfering with any other variable?

Reply:
It's shadowing an outer variable (I got carried away)

Reply:
Want to do another PR for this? Or include it in this one?

Reply:
Let's include in this one, I don't want to be responsible for going above 1.5°c of global warming due to an irresponsible use of heavy duty primers :smile:

Reply:
If my code is correct they shouldn't on a change that only affects `tests/**` but I think this is fine to include here indeed 😄